spec/unit/hanami/events/matcher_spec.rb

```diff
@@ -0,0 +1,59 @@
+RSpec.describe Hanami::Events::Matcher do
+  let(:matcher) { described_class.new(pattern) }
+
+  context 'when pattern match only one event' do
+    let(:pattern) { 'user.created' }
+
+    it { expect(matcher.match?('user.created')).to be true }
```

Comment:
sorry, but:
1. I don't like `shared_examples` because this is not an obvious way of testing. Also, this abstraction doesn't solve our problems.
2. We can create matcher for this instead. 
3. I'm not sure that we should work on it now.